src/components/Checkbox.tsx

```diff
@@ -60,10 +60,9 @@ const CheckboxInput = styled.input`
   }
 `;
 
-const Checkbox = forwardRef(
-  (props: HTMLProps<HTMLInputElement>, ref: Ref<HTMLInputElement>) => (
-    <CheckboxInput {...props} type="checkbox" ref={ref} />
-  ),
-);
+const Checkbox = forwardRef((props: HTMLProps<HTMLInputElement>, ref: Ref<HTMLInputElement>) => (
+  // eslint-disable-next-line
```

Comment:
I think we can disable `jsx-props-no-spreading`rule globally  in eslintrc.js.

Reply:
good idea!